### ISSUE TITLE
SafeAreaView behavior unexpected on non iPhone X devices

### ISSUE THREAD

**Issue by embpdaniel** [react-native-community/react-native-safe-area-view#37]
Hi, I have an app where the iPhoneX status bar overlaps some of my top menu content. To add padding to the area, I added a SafeAreaView component to the bar. It looks perfect on an iPhone X in simulator, but when I run this on a simulated device that is not an  iPhone X, the padding remains, and now I have too much space between my menu items and the status bar.

I was thinking that the component would detect if it was on iPhone X or not, and properly adjust the height, so that it looks perfect on all devices. Is this not the case? I am using the SafeAreaView library v0.9.0

Here's what my code looks like:

```javascript
import SafeAreaView from 'react-native-safe-area-view';
...
<SafeAreaView>
    <NavigationBar
        title={this._title()}
        leftButton={this._leftButton()}
        rightButton={this._rightButton()}
        style={styles.navBar}
    />
</SafeAreaView>
```

Let me know, thanks

**Comment by kevbost** [comment]
I just went through this same issue.  I built the app to look great in the simulator, but when I tested on an actual iPhoneX device everything was wonko.

I manually stripped out all of the `SafeAreaView`'s from `react-native` except for the one surrounding the `Root` component in `App.js`, and used the one from `react-navigation` instead, then had to do some more tweaking inside of some components where `headerMode: 'none'` was set.

The side effect to making it correct on a real device is it going back to being wonko in the simulator -- and unfortunately I don't have a solution for that.

EDIT: I'm aware that this is more of a `react-navigation` problem, but this issue in particular is what I worked through -- differences between the Simulator and a real device.
